app/qmlV2/component/MMAbstractEditor.qml

```diff
@@ -0,0 +1,181 @@
+        checked: root.checkboxChecked
+      }
+      Text {
+        id: titleItem
+
+        width: parent.width - checkbox.width - titleRow.spacing
+        font: StyleV2.p6
+        wrapMode: Text.WordWrap
+      }
+    }
+
+    Item {
+      height: 50 * __dp
+      anchors.left: parent.left
+      anchors.right: parent.right
+
+      Rectangle {
+        id: background
+
+        width: parent.width
+        height: parent.height
+        border.color: root.hasFocus ? ( errorMsg.length > 0 ? StyleV2.negativeColor : warningMsg.length > 0 ? StyleV2.warningColor : StyleV2.forestColor ) : StyleV2.transparentColor
+        border.width: 2 * __dp
+        color: root.bgColor
+        radius: 12 * __dp
+      }
+
+      Row {
+        height: parent.height
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.leftMargin: root.spacing
+        anchors.rightMargin: root.spacing
+
+        Item {
+          id: leftActionContainer
+
+          property bool actionAllowed: leftActionContainer.children.length > 1 && (leftActionContainer.children[1]?.visible ?? false)
+
+          height: parent.height
+          width: actionAllowed ? height/2 : 0
+
+          Rectangle {
+            color: StyleV2.transparentColor
+            anchors.centerIn: parent
+            width: leftActionContainer.actionAllowed ? parent.width + root.spacing/2 : 0
+            height: parent.height
+
+            MouseArea {
+              anchors.fill: parent
+              onReleased: root.leftActionClicked()
+            }
+          }
+        }
+
+        Item {
+          id: contentContainer
+
+          height: parent.height
+          width: parent.width - (leftActionContainer.actionAllowed ? leftActionContainer.width : 0) - (rightActionContainer.actionAllowed ? rightActionContainer.width : 0) //- 2 * root.spacing - ( leftActionContainer.actionAllowed ? root.spacing : 0 )
+
+          MouseArea {
+            anchors.fill: parent
+
+            onClicked: {
+              root.contentClicked()
+            }
+          }
+        }
+
+        Item {
+          id: rightActionContainer
+
+          property bool actionAllowed: rightActionContainer.children.length > 1 && (rightActionContainer.children[1]?.visible ?? false)
+
+          height: parent.height
+          width: actionAllowed ? height/2 : 0
+
+          Rectangle {
+            color: StyleV2.transparentColor
+            anchors.centerIn: parent
+            width: rightActionContainer.actionAllowed ? parent.width + root.spacing/2 : 0
+            height: parent.height
+
+            MouseArea {
+              anchors.fill: parent
+              onReleased: root.rightActionClicked()
+            }
+          }
+        }
+      }
+    }
+
+    Item {
+      id: messageItem
+
+      width: parent.width
+      height: msgRow.height
+
+      Row {
+        id: msgRow
+
+        spacing: 4 * __dp
+
+        MMIcon {
+          id: msgIcon
+
```

Comment:
I think you are missing `anchors.top: parent.top` here as this way you won't ensure keeping the error icon on top when the text has multiple lines
![image](https://github.com/MerginMaps/mobile/assets/22449698/0dba22f4-6eda-4fc0-b551-46810c99e323)